Sources/System/Sockets/SocketAddress.swift

```diff
@@ -21,34 +19,56 @@ public struct SocketAddress {
   }
 
   public init(_ buffer: UnsafeRawBufferPointer) {
-    precondition(buffer.count >= MemoryLayout<CInterop.SockAddr>.size)
```

Comment:
Is the precondition no longer worth checking?

Reply:
Yes! SocketAddress now supports buffers of any size. The capacity will never be shorter than what fits in inline storage, so this'll not lead to problems when accessing `family`.